src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -349,11 +372,12 @@ private void getChanges2(ChangeEventSourceContext context,
         // Note that the entry will be created only when a BEGIN block is encountered.
         Map<String, Integer> recordsInTransactionalBlock = new HashMap<>();
 
-        LOGGER.debug("The init tabletSourceInfo is " + offsetContext.getTabletSourceInfo());
+        LOGGER.info("The init tabletSourceInfo is " + offsetContext.getTabletSourceInfo());
```

Comment:
The same information is logged twice. First time in line 359 above.

Reply:
Also, do we need this information at the info level log? If not, then we should keep it at the debug level only.

Reply:
Update: This log statement meant to print `offsetContext.getTabletSourceInfo()` - one before updating the `offsetContext` and another after updating it. Updating the PR with the necessary changes.